guide/english/certifications/javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-with-the-rest-operator-to-reassign-array-elements/index.md

```diff
@@ -42,28 +42,35 @@ function removeFirstTwo(list) {
 }
 ```
 
-## Hint 3 
 
-Exclude the first two elements of the `arr` array with `,,`.
+## Spoiler Alert - Solution Ahead!
+You can use random variables to omit the first two values:
 
 ```javascript
 function removeFirstTwo(list) {
   "use strict";
   // change code below this line
-  const [,,...arr] = list; // change this
+  const [a, b, ...arr] = list; 
```

Comment:
This solution does not work:
![image](https://user-images.githubusercontent.com/34807532/55003746-f6a7f280-4ffe-11e9-8095-b1b401563a8a.png)

Reply:
@thecodingaviator Technically, that solution should work, but the current tests are looking for something very specific.  I will create an issue where we can discuss this further.  For now, let's hold off on not allowing this change, but also block it until we get consensus on how best to modify the existing test.

Reply:
Sounds okay

Reply:
I believe something like the following for the last test would work fine:
```yml
  - text: Destructuring on <code>list</code> should be used.
    testString: assert(code.replace(/\s/g, '').match(/\[(([a-z]\w*)?,){1,}\.\.\.arr\]=list/gi));
```

Reply:
Looks good to me 👍 .